internal/perf/perf.go

```diff
@@ -162,25 +163,29 @@ func (pr *perfRunner) eventLoop() (err error) {
 			var event fftypes.EventDelivery
 			json.Unmarshal(msgBytes, &event)
 
-			var workerId int
+			var workerID int
 
 			switch event.Type {
 			case fftypes.EventTypeBlockchainEventReceived:
-				workerId, err = pr.GetBlockchainEventValue(pr.cfg.Node, event.Reference.String())
+				if event.BlockchainEvent == nil {
+					log.Errorf("\nBlockchain event not found --- Event ID: %s\n\t%d --- Ref: %s", event.ID.String(), event.Reference)
+				}
```

Comment:
Does the if statement need to return if entered?

Reply:
I think that would kill the test, or at least this worker. Is that desirable or not?

Reply:
If it proceeds, won't there be a nil pointerin the line below?